src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -64,6 +65,7 @@ public void Add(Cookie cookie)
             {
                 throw new ArgumentNullException(nameof(cookie));
             }
+            ++m_version;
```

Comment:
> it wouldn't be ported to Core.

It absolutely would be.  We shouldn't be adding new netfx-only additions to existing types.